### PR TITLE
[IOTDB-6061] Fix the instability failure caused by initServer in IoTConsensus UT not binding to the corresponding port 

### PR DESCRIPTION
**Pull request by OneSizeFitsQuorum**

### CODE REVIEW

File: iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/iot/ReplicateTest.java [hunk 1]
```diff
@@ -115,8 +115,7 @@ public void changeConfiguration(int i) {
         Files.delete(configurationPath);
       }
       Files.move(tmpConfigurationPath, configurationPath);
-    } catch (IOException e) {
-      logger.error("Unexpected error occurs when persisting configuration", e);
+    } catch (IOException ignored) {
```
Comment (wangchao316) [inline iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/iot/ReplicateTest.java:118]: haha , xinyu, this is a code smell

**Review by wangchao316 — COMMENTED** [review]
L

File: iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/iot/ReplicateTest.java (hunk 1 above)
Comment (OneSizeFitsQuorum) [reply iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/iot/ReplicateTest.java:118]: Thanks for review! Fixed

**Review by wangchao316 — APPROVED** [review]

**Comment by OneSizeFitsQuorum** [comment]
seems useless

**Pull request closed without merging**